Header header
float64 distance
uint8 auv_id